src/build/optimize-streams.ts

```diff
@@ -30,6 +30,11 @@ export type FileCB = (error?: any, file?: File) => void;
 export type CSSOptimizeOptions = {
   stripWhitespace?: boolean;
 };
+export interface OptimizeOptions {
+  htmlMinify?: HTMLMinifierOptions;
```

Comment:
I wanted to pass this subset of build options to getOptimizeStreams() to simplify the interface. But I can match the nested structure from BuildOptions if you prefer.